package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@clutter/wt",
-  "version": "2.1.0",
+  "version": "3.0.0-alpha.1",
```

Comment:
Do you want to just mark as `3.0.0`? What's our naming convention for this?

Reply:
I'm in favor of leaving as an alpha because I'd really really like to remove the image fallback which would technically qualify as a breaking change IMO.

Reply:
Fair point (and seems a bit strange to go from v3 to v4 in a few days).